modmesh/pylibmgr.py

```diff
@@ -27,10 +27,35 @@
 import sys
 import os
 import time
+import importlib.abc
+import importlib.machinery
 
 lib_path = {}
 
 
+class mm_path_finder(importlib.abc.MetaPathFinder):
+    def find_spec(self, lib_name, path, target=None):
+        if lib_name in lib_path:
+            _ = os.path.abspath(lib_path[lib_name])
+            pkg_path = os.path.join(_, lib_name)
+            init_path = os.path.join(pkg_path, '__init__.py')
+
+            if not os.path.exists(init_path):
+                return None
+
+            loader = importlib.machinery.SourceFileLoader(lib_name, init_path)
+            spec = importlib.machinery.ModuleSpec(
+                                                  lib_name,
```

Comment:
The indentation makes too many lines.  Could you please change it to be more compact:

```python
spec = importlib.machinery.ModuleSpec(
        lib_name, loader, origin=init_path, is_package=True)
```

Reply:
Done !